site-src/geps/overview.md

```diff
@@ -0,0 +1,123 @@
+against them. At this stage, the GEP should be targeting the "Implementable"
+stage.
+
+### 4. Implement the GEP as "Experimental"
+With the GEP marked as "Implementable", it is time to actually make those
+proposed changes in our API. In some cases, these changes will be documentation
+only, but in most cases, some API changes will also be required. It is important
+that every new feature of the API is marked as "Experimental" when it is
+introduced. Within the API, we use `<gateway:experimental>` tags to denote
+experimental fields.
+
+Before these changes are released, they will also need to be documented.
+GEPs that have not been both implemented and documented before a release
+cut off will be excluded from the release.
+
+### 5. Graduate the GEP to "Standard"
+Once this feature has met the [graduation criteria](/concepts/versioning/#graduation-criteria), it is
+time to graduate it to the "Standard" channel of the API. Depending on the feature, this may include
+any of the following:
+
+1. Graduating the resource to beta
+2. Graduating fields to "standard" by removing `<gateway:experimental>` tags
+3. Graduating a concept to "standard" by updating documentation
+
+## Status
+
+Each GEP has a status field that defines it's current state. Most GEPS will
+proceed through the following states:
+
+* **Provisional:** The goals described by this GEP have consensus but
+  implementation details have not been agreed to yet.
+* **Implementable:** The goals and implementation details described by this GEP
+  have consensus but have not been fully implemented yet.
+* **Experimental:** This GEP has been implemented and is part of the
+  "Experimental" release channel. Breaking changes are still possible.
+* **Standard:** This GEP has been implemented and is part of the
+  "Standard" release channel. It should be quite stable.
+
+Although less common, some GEPs may end up in one of the following states:
+
+* **Deferred:** We do not currently have bandwidth to handle this GEP, it
+  may be revisited in the future.
+* **Rejected:** This proposal was considered by the community but ultimately
+  rejected.
+* **Replaced:** This proposal was considered by the community but ultimately
+  replaced by a newer proposal.
+* **Withdrawn:** This proposal was considered by the community but ultimately
+  withdrawn by the author.
+
+## Format
+
+GEPs should match the format of the template found in [GEP-696](/geps/gep-696).
+
+## Out of scope
+
+What is out of scope: see [text from KEP][kep-when-to-use]. Examples:
+
+* Bug fixes
+* Small changes (API validation, documentation, fixups). It always
+  possible that the reviewers will determine a "small" change ends up
+  requiring a GEP.
+
+## How much additional work is this?
+
+* Need to copy out material into a `.md` document for future
+  reference and keep it up to date.
+* Need to keep GEP up to date with current references to shared doc.
+* Some amount of overhead merging GEPs into the repo.
+
```

Comment:
This section feels deprecated now - this is the established process now, rather than a proposed alternative to some other process.
```suggestion
```

Reply:
Removed, thanks!